paddle/fluid/inference/tensorrt/convert/io_converter.h

```diff
@@ -25,43 +26,57 @@ namespace tensorrt {
 using framework::LoDTensor;
 
 /*
- * Convert Input from Fluid to an Engine.
- * TensorRT's ITensor follows row major, NCHW. Fluid is also row major, so in
- * most cases just need to copy the data.
+ * Convert Input from Fluid to TensorRT Engine.
+ * Convert Output from TensorRT Engine to Fluid.
+ *
+ * Note that TensorRT's ITensor follows row major, NCHW. Fluid is also row
+ * major,
+ * so in the default case just need to copy the data.
  */
-class EngineInputConverter {
+class EngineIOConverter {
```

Comment:
Add comments about how to register a convter from fluid to TRT or from TRT to fluid.

For example, a converter for a special Op called `x` that from fluid to TRT will register as `Fluid(x)->TRT`.

Or a special Layer in TRT called `y` that convert output from TRT to fluid will register as `TRT(y)->Fluid`.